src/ragapp/Makefile

```diff
@@ -60,3 +61,7 @@ dev:
 	poetry run python main.py & \
 	pnpm --prefix ./admin-ui run dev & \
 	wait
+
+mypy:
```

Comment:
do we trigger mypy in CI?

Reply:
Ah, sorry. We don't trigger mypy in the CI as it's not working for RAGapp codebase now. I tried to add mypy to RAGapp but it had conflict errors that i couldn't fix it quickly. I'll work on this later.